ts/src/test/static/request/bitget.json

```diff
@@ -933,11 +933,23 @@
             {
                 "description": "Spot cancel all orders",
                 "method": "cancelAllOrders",
-                "url": "https://api.bitget.com/api/v2/spot/trade/cancel-symbol-order",
```

Comment:
aren't we removing this test?

Reply:
@carlosmiei oops, the test was removed to the next.

Reply:
oh ok, but still tests are broken going to check them